public/js/guests/controllers/rsvpController.js

```diff
@@ -0,0 +1,37 @@
+define([
+    '../module',
+    '../namespace',
+], function (module, namespace) {
+    'use strict';
+
+    var name = namespace + ".RsvpController";
+
+    module.controller(name, ['$scope', '$window', namespace + '.GuestServices', function ($scope, $window, GuestServices) {
+        $scope.guest = {};
+        $scope.attending = {};
+        $scope.attempts = 0;
+
+        $scope.validate = function (rsvpCode) {
+           GuestServices.get({ rsvp : rsvpCode }, function(guest) {
```

Comment:
You should do something on error. Probably just "Something went wrong, [email us](mailto: etc)" 

This can be handled in a later change but what if someone has already rsvp'd? I need to be notified somehow. I'm thinking either a mailto link telling them to explain it, or give them the same information and I'll need some kind of update information under /guests.